Userland/Libraries/LibWeb/HTML/EmbedderPolicy.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2024, Jamie Mansfield <jmansfield@cadixdev.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Optional.h>
+#include <AK/String.h>
+#include <AK/StringView.h>
+
+namespace Web::HTML {
+
+// https://html.spec.whatwg.org/multipage/browsers.html#embedder-policy-value
+enum class EmbedderPolicyValue {
+    UnsafeNone,
+    RequireCorp,
+    Credentialless,
+};
+
+StringView to_string(EmbedderPolicyValue);
+Optional<EmbedderPolicyValue> from_string(StringView);
```

Comment:
This feels like a strange thing to put in the `Web::HTML` namespace.

Instead of `from_string(StringView)`, perhaps it should be called `embedder_policy_value_from_string(StringView)`?

Reply:
Silly me, will correct this evening :)